cockatrice/src/player.cpp

```diff
@@ -585,18 +588,24 @@ void Player::retranslateUi()
     rfgMenu->setTitle(tr("&Exile"));
     
     if (local) {
-        aMoveHandToTopLibrary->setText(tr("Move to &top of library"));
-        aMoveHandToBottomLibrary->setText(tr("Move to &bottom of library"));
-        aMoveHandToGrave->setText(tr("Move to &graveyard"));
-        aMoveHandToRfg->setText(tr("Move to &exile"));
-        aMoveGraveToTopLibrary->setText(tr("Move to &top of library"));
-        aMoveGraveToBottomLibrary->setText(tr("Move to &bottom of library"));
-        aMoveGraveToHand->setText(tr("Move to &hand"));
-        aMoveGraveToRfg->setText(tr("Move to &exile"));
-        aMoveRfgToTopLibrary->setText(tr("Move to &top of library"));
-        aMoveRfgToBottomLibrary->setText(tr("Move to &bottom of library"));
-        aMoveRfgToHand->setText(tr("Move to &hand"));
-        aMoveRfgToGrave->setText(tr("Move to &graveyard"));
+        moveHandMenu->setTitle(tr("&Move hand to"));
```

Comment:
Minor tweak, but can you make the upper level menus "Move foo to..." and capitalize the submenu entries? After that we should be good to merge.